cartofacade/capi_test.go

```diff
@@ -49,7 +49,7 @@ func testAddSensorReading(t *testing.T, vc Carto, sensor, pcdPath string, timest
 	err = pointcloud.ToPCD(pc, buf, pcdType)
 	test.That(t, err, test.ShouldBeNil)
 
-	err = vc.addSensorReading(sensor, buf.Bytes(), timestamp)
+	err = vc.addSensorReading("mysensor", buf.Bytes(), timestamp)
```

Comment:
fixes error from go vet which detected that the `sensor` parameter passed to `testAddSensorReading` was always the same.